src/domain_bridge/wait_for_graph_events.hpp

```diff
@@ -199,6 +199,11 @@ class WaitForGraphEvents
     // If there are no publishers, return an empty optional
     if (num_endpoints < 1u) {
       return {};
+    } else {
+      // If we found at least one publisher, wait for a bit (more publishers may become available)
+      std::this_thread::sleep_for(std::chrono::milliseconds(300));
+      endpoint_info_vec = node.get_publishers_info_by_topic(topic);
```

Comment:
Rather than querying for endpoints twice, I guess it's equivalent to first sleep and then query once. I.e. I think it's simpler to (optionally) sleep before calling `get_publishers_info_by_topic` above.

Reply:
Yeah, that does seem a bit simpler. Done in 05b6497